uktrade/data-hub-frontend
# Add linting rule to prevent duplicate imports

## Description of change

**Note** - This is pointing towards the #6110 branch to avoid conflicts

Another issue I've noticed recently in PR review is that occasionally we are adding duplicate imports. I've enabled the [no-duplicate-imports rule](https://eslint.org/docs/latest/rules/no-duplicate-imports) in ESLint to catch these.

## Test instructions
Attempting to add a duplicate import should now raise an error.

<img width="599" alt="Screenshot 2023-09-27 at 17 12 26" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/d465ce7c-438e-47f5-8685-6a36968cc5cf">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
